pkg/testing/helm/helm_test.go

```diff
@@ -92,6 +93,39 @@ func (h *HelmTestSuite) TestUpgrade(c *C) {
 	c.Assert(err, IsNil)
 }
 
+func (h *HelmTestSuite) TestDryRunInstall(c *C) {
+	defer func() {
+		h.helmApp.dryRun = false
+	}()
+	c.Log("Installing kanister release - Dry run")
+	h.helmApp.dryRun = true
+	_, err := h.helmApp.Install()
+	c.Assert(err, IsNil)
+}
```

Comment:
I don't think this test is providing a lot of value. what exactly we are testing in here? If not really helpful we can remove this.

Reply:
Removed